Constitution.txt

```diff
@@ -1,490 +1,264 @@
+8: In all criminal prosecutions, the accused shall enjoy the right to a speedy and public trial, by an impartial jury of the State and district wherein the crime shall have been committed, which district shall have been previously ascertained by law, and to be informed of the nature and cause of the accusation; to be confronted with the witnesses against him; to have compulsory process for obtaining witnesses in his favor, and to have the Assistance of Counsel for his defense, such Counsel to be provided for by the State and district in which he is accused, in the event that he cannot procure Counsel for himself.
 
-2. The Congress shall have power to enforce this article by appropriate legislation.   ratified #24
+9: In Suits at common law, where the value in controversy shall exceed the value of five days' labor of any involved Party, the right of trial by jury shall be preserved, and no fact tried by a jury, shall be otherwise re-examined in any Court of these United States, than according to the rules of the common law.
 
-Amendment XXV  affects 9 (Amendment 25 - Presidential Vacancy, Disability, and Inability)
-1: In case of the removal of the President from office or of his death or resignation, the Vice President shall become President.
+10: Excessive bail shall not be required, nor excessive fines imposed, nor cruel or unusual punishments inflicted.
 
-2: Whenever there is a vacancy in the office of the Vice President, the President shall nominate a Vice President who shall take office upon confirmation by a majority vote of both Houses of Congress.
+11: Neither slavery nor involuntary servitude, except as a punishment for crime whereof the party shall have been duly convicted, shall exist within these United States, or any place subject to their jurisdiction; and no Person shall be considered the Property of any other Person.
 
-3: Whenever the President transmits to the President pro tempore of the Senate and the Speaker of the House of Representatives his written declaration that he is unable to discharge the powers and duties of his office, and until he transmits to them a written declaration to the contrary, such powers and duties shall be discharged by the Vice President as Acting President.
+Article VII (Article 7 - Definition of Citizenship)
+1: All persons naturalized in these United States, born to natural Citizens of these United States, or resident in and subject to any State joining the Union at time of joining, are Citizens of these United States and of the State wherein they reside. No State shall make or enforce any law which shall abridge the privileges or immunities of citizens of these United States; nor shall any State deprive any person of life, liberty, or property, without due process of law; nor deny to any Person within its jurisdiction the equal protection of the laws.
```

Comment:
Here I update the definition of Citizen so as not to include those born on US soil, but not to US citizens. That text of the 14th Amendment was made with the purpose of making former Slaves into Citizens, and was right to do so. However, that one-time impact has been made, and the text now serves as a detriment, drawing citizens of other countries to make potentially dangerous journeys while pregnant, in the hope that their child will become a Citizen of the US by a legal technicality.

Naturalization of people previously unknown to the US Government must be a formal process. If the Congress decides that this formal process can be as simple as a birth certificate in a US hospital, that will be acceptable, but such a policy should not arise from a court decision on implication of text in the Constitution.